internal/idler/user_idler.go

```diff
@@ -110,6 +114,15 @@ func (idler *UserIdler) checkIdle() error {
 		idler.logger.Warnf("idler disabled for user %s - skipping", idler.user.Name)
 		return nil
 	}
+	userDisableStatus, err := idler.checkUserStatus()
```

Comment:
shouldn't  this be handled in the controller itself? why even create a user-idler (if it hasn't already been created) if the idler is disabled for that user?